packages/clarity-decode/src/layout.ts

```diff
@@ -20,12 +20,12 @@ export function decode(tokens: Data.Token[]): LayoutEvent {
         case Data.Event.Region:
             let regionData: Layout.RegionData[] = [];
             for (let i = 2; i < tokens.length; i += 3) {
-                // For backward compatibility since 0.6.4, we extract visibility signal from the "box" field if it is present
-                let legacy = Array.isArray(tokens[i+1]) && (tokens[i+1] as number[]).length === 5;
+                // For backward compatibility since 0.6.12, map visibility to state
+                let legacy = typeof tokens[i + 1] === Constant.Number;
                 let region: Layout.RegionData = {
                     id: tokens[i] as number,
-                    visible: legacy ? tokens[i + 1][4] as number : tokens[i + 1] as number,
-                    region: tokens[i + 2] as string
+                    name: legacy ? tokens[i + 2] as string : tokens[i + 1] as string,
```

Comment:
>tokens [](start = 35, length = 6)

why not keep it consistent - i + 1 is state, i + 2 is name ?

Reply:
Good point. will update.

---
In reply to: [643631103](https://github.com/microsoft/clarity/pull/137#discussion_r643631103) [](ancestors = 643631103)